### PR TITLE
Component test

### PR DESCRIPTION
**Pull request by Jakub-Fajkus**

### CODE REVIEW

**Comment by tomasedl** [comment]
just add please package for dao in test

**Comment by Jakub-Fajkus** [comment]
@tomasedl moved to package, thanks.. anything else?

File: src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Car.java [hunk 1]
```diff
@@ -71,6 +71,8 @@ public void setDriver(Driver driver) {
 
     public void addComponent(Component component) {
         this.components.add(component);
+
+        component.setCar(this);
```
Comment (khecova) [inline src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Car.java:75]: budem to tady teda resit? ... potom by se melo vyresit i mazani v removeComponent, at to odnastavi, pokud to ma byt nejak konzistentni

Comment (Jakub-Fajkus) [reply src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Car.java:75]: smazu

Comment (Jakub-Fajkus) [reply src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Car.java:75]: a vyresime komplexne pozdeji

**Review by khecova — APPROVED** [review]
nice work 👍

**Review by andrlos — APPROVED** [review]
good job

**Review by tomasedl — APPROVED** [review]

**Pull request merged**